charms/worker/k8s/src/token_distributor.py

```diff
@@ -308,6 +309,7 @@ def active_nodes(self, relation: ops.Relation):
         return {
             self.charm.model.get_unit(str(u)): data
             for u, data in relation.data[self.charm.app].items()
+            if re.match(r"k8s(-worker)?/\d+", u)
```

Comment:
Completely personal: I'm bad at reading and understanding regex right away, maybe we can add a comment with an example in front of it.
```suggestion
            if re.match(r"k8s(-worker)?/\d+", u) # k8s/123, k8s-worker/123
```

Reply:
Sure, let's add an explanation to the docstring instead.

Reply:
also, building re's in a loop is considered bad practice.  

can you consider making `re.compile(r"k8s(-worker)?/\d+")` as a constant `UNIT_RE` and then running the match in this loop?

Reply:
TIL